openapi/code/method.go

```diff
@@ -179,6 +179,30 @@ func (m *Method) IsJsonOnly() bool {
 	return m.Operation.JsonOnly
 }
 
+func (m *Method) CanSetRequiredFieldsFromJson() bool {
```

Comment:
Could you add a comment to very clearly state what we expect to hold here?

All branches below in isolation I understand, but in aggregate can't tell what the implication is of this condition being either true or false.